Added clarification about the PR workflow (#259)
redhat-documentation/supplementary-style-guide: CONTRIBUTING.md

   Typically the previous command gives the URL to open a pull request. If not, you can open one from the [Pull requests](https://github.com/redhat-documentation/supplementary-style-guide/pulls) tab of the GitHub UI.

After you submit a pull request, it will be reviewed by repository administrators. 

> When the PR has at least three approvals, the PR can then be merged to the _main_ branch by one of the repo administrators.

Reviewer: This is a nitpick, but should it `main` be in backticks rather than in italics?

Reviewer 2: I agree with Brian about the backticks. We should also rewrite both sentences to remove the passive voice?

Reviewer: @bredamc +1 to revising to remove the passive voice.

Author: @briburt & @bredamc - Thank you both for flagging :+1:  Both issues are now fixed. :balance_scale:

I'm asking myself why I ignored the prominent yellow and blue Vale linter warnings in my IDE :thinking:

It then got me thinking: Have we considered integrating the Vale linter with the vale-at-red-hat style rules into the PR workflow  of this repo? If it ran silently in the background in non-blocking mode, it could also double up to help improve the accuracy of the vale-at-red-hat rule set. But perhaps this is a suggestion for another channel...

> When the PR has been reviewed and approved by at least three `supplementary-style-guide` reviewers, the PR will be merged to the _main_ branch by a repository administrator.

Reviewer 2: Should we specify that the PR must be approved by 3 repo admins, rather than any 3 SSG reviewers?

   Typically the previous command gives the URL to open a pull request. If not, you can open one from the [Pull requests](https://github.com/redhat-documentation/supplementary-style-guide/pulls) tab of the GitHub UI.

After you submit a pull request, at least three repository administrators must review the content changes. 

> When your PR has three approvals, one of the repository administrators will merge your PR to the `main` branch.

Reviewer 2: ```suggestion
When your PR has three approvals, a repository administrator will merge your PR to the `main` branch.
```
